sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5553,9 +5553,9 @@ def test_issue_7180():
 
 
 def test_pretty_Complement():
-    assert pretty(S.Reals - S.Naturals) == '(-oo, oo) \ Naturals()'
+    assert pretty(S.Reals - S.Naturals) == 'Interval(-oo, oo, True, True) \ S.Naturals'
```

Comment:
@SagarB-97 the easiest way is just to leave the pretty printer's behaviour unaffected.

Otherwise, there is a symbol for real numbers: http://www.fileformat.info/info/unicode/char/211d/index.htm

If use_unicode=False, I would print it either as `R` or as `(-oo, oo)`.